manual/C/ch_Finance-Quote.xml

```diff
@@ -195,7 +195,8 @@ SYNOPSIS
         </simpara>
       </footnote>
       can be used to fetch the current quotes of your stocks and write them directly into your
-      &app;-data file.
+      &app;-data file without starting &app; itself. Thus an automatic, 
+      regular updating of the price database is possible.
     </para>
 
     <para>The file specified <filename><replaceable>${HOME}/gnucash-filename</replaceable></filename> will
```

Comment:
`<filename><replaceable>${HOME}/gnucash-filename</replaceable></filename>` create an entity for this and replace similar occurrences. That helps also, if we find a better solution.